lib/llm/providers/anthropic.rb

```diff
@@ -43,5 +43,17 @@ def auth(req)
     def response_parser
       LLM::Anthropic::ResponseParser
     end
+
+    def transform_prompt(prompt)
```

Comment:
We should add references to Provider docs, Yard will follow them when generating docs. Eg:
```
##
# @param prompt (see LLM::Provider#transform_prompt)
# @return (see LLM::Provider#transform_prompt)